.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+# Configuration for GitHub Action `.github/workflows/pull-request-labeler.yml`.
+# See https://github.com/marketplace/actions/labeler for more information.
+
+# Only documentation changes
+docs:
+  - all: [ '**/*.md' ]
+
+# Only test changes
+dev/testing:
+  - all: [ '{spec/**,integration/**}' ]
+
+# Changes to Profiling
+profiling:
+  - any: [ 'lib/datadog/profiling/**' ]
+
+# Changes to CI-App
+ci-app:
+  - any: [ 'lib/datadog/ci/**' ]
```

Comment:
Thanks for giving my fork a go folks! Does this glob work the same as `lib/datadog/ci/**/*`? I can update docs in https://github.com/actions/labeler/pull/316 if so.

Reply:
Thank you for the changes, @kachkaev! Couldn't use the GH Action without it! 🙇

Yes, it works the same.
```js
minimatch("root/a/b/c", "root/**/*")
// true
minimatch("root/a/b/c", "root/**")
// true 
```